### PR TITLE
#2542 Manager's E2E test should give the cluster name a prefix

### PR DESCRIPTION
**Pull request by konekoya**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #2542)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

File: ohara-manager/scripts/handleE2eServices.js [hunk 1]
```diff
@@ -80,22 +79,22 @@ exports.createServices = async ({
...
     await createNode({ apiRoot, nodeHost, nodePort, nodeUser, nodePass });
 
     const zookeeperUrl = `${apiRoot}/zookeepers`;
     const brokerUrl = `${apiRoot}/brokers`;
+    const workerUrl = `${apiRoot}/workers`;
 
     await axios.post(zookeeperUrl, {
-      clientPort: commonUtils.randomPort(),
-      electionPort: commonUtils.randomPort(),
-      peerPort: commonUtils.randomPort(),
```
Comment (konekoya) [inline ohara-manager/scripts/handleE2eServices.js]: These fields are now can be safely generated by backend services

File: ohara-manager/build.gradle [hunk 2]
```diff
@@ -109,8 +109,11 @@ task e2e {
 
         if(!project.hasProperty('ohara.manager.e2e.nodePass')) throw new IllegalArgumentException("ohara.manager.e2e.nodePass is required")
         String nodePass = project.getProperty('ohara.manager.e2e.nodePass')
+
+        if(!project.hasProperty('ohara.it.container.prefix')) throw new IllegalArgumentException("ohara.it.container.prefix is required")
+        String prefix = project.getProperty('ohara.it.container.prefix')
```
Comment (konekoya) [inline ohara-manager/build.gradle]: @chia7712 Does this look good to you :)

File: ohara-manager/client/cypress/support/retryOnFail.js [hunk 3]
```diff
@@ -21,5 +21,5 @@ const integrationFolderName = Cypress.config('integrationFolder')
 // Enable cypress to retry on fail end-to-end test
 if (integrationFolderName === 'e2e') {
   require('cypress-plugin-retries');
-  Cypress.env('RETRIES', 5);
+  Cypress.env('RETRIES', 3);
```
Comment (konekoya) [inline ohara-manager/client/cypress/support/retryOnFail.js]: Decrease the retry times so we won't have to wait super long for a failed build...

File: ohara-manager/scripts/handleE2eServices.js [hunk 4]
```diff
@@ -122,6 +118,26 @@ exports.createServices = async ({
       condition: () => isServiceReady(`${brokerUrl}/${brokerClusterName}`),
     });
 
+    await axios.post(workerUrl, {
```
Comment (konekoya) [inline ohara-manager/scripts/handleE2eServices.js]: We're relying on worker API's tags to obtain the services we're going to delete when a test run finished. So we need to start a worker as well here.

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by chia7712** [comment]
借我測一下api

**Comment by chia7712** [comment]
retry manager-api

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Pull request merged**